src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -503,7 +503,6 @@ function ReportPreview({
                             images={lastThreeReceipts}
                             total={allTransactions.length}
                             size={CONST.RECEIPT.MAX_REPORT_PREVIEW_RECEIPTS}
-                            onPress={openReportFromPreview}
```

Comment:
This component is used in many places, so we should create a new prop `shouldDisableOnPress`, to prevent the action when the parent is already handling it, as seen in `MoneyRequestPreviewContent` to avoid regression.

Reply:
Ah, this action is also handled by [the parent](https://github.com/bernhardoj/Expensify/blob/main/src/components/ReportActionItem/ReportPreview.tsx#L492)  container so it's fine to remove it.